lib/registries/addon/overview/route.ts

```diff
@@ -76,23 +79,68 @@ export default class Overview extends GuidRoute {
                 ),
                 institution: (institutions as SparseModel[]).map(institution => institution.name as string),
             };
-            const headTags = [...this.metaTags.getHeadTags(metaTagsData)];
+
+            const metaTags: HeadTagDef[] = this.metaTags.getHeadTags(metaTagsData);
```

Comment:
Might as well revert the rename because the `metaTags` array contains not only `<meta>` tags, but also other head tags.